tests/test_vtk.py

```diff
@@ -98,6 +100,25 @@ def test_pluto_two_units_override(vtk_file_with_units):
     assert_allclose_units(ds.mass_unit, expect_mass)
 
 
+def test_missing_inifile(tmp_path):
+    tmpdir = tmp_path / "missing_inifile"
+    shutil.copytree(
+        Path(__file__).parent / "data" / "pluto_disk_planet",
+        tmpdir,
+        ignore=shutil.ignore_patterns("*.ini"),
+    )
+    with pytest.warns(
+        UserWarning, match=r"The inifile is missing for unit definitions. *"
+    ):
+        yt.load(tmpdir / "data.0010.vtk")
+
+
+def test_incorrect_inifile():
+    file_dir = Path(__file__).parent / "data" / "pluto_disk_planet"
+    with pytest.warns(UserWarning, match=r"Cannot get the value of *"):
```

Comment:
Clarification: the `match` argument expects a regexp pattern. In particular this means
- `' *'` is interpreted as "any number of `' '`characters" (I assume you meant "anything next should match")
- matching a string only means that the *start* of the string fits the patterns, so we don't actually need to fill the whole message or do anything special to express that the rest of the string can be ignored.
```suggestion
    with pytest.warns(UserWarning, match=r"Cannot get the value of"):
```

Reply:
Oh I thought it would pass only when the whole warning message is matched.

> I assume you meant "anything next should match"

That's true, I thought it is like that in shell.